plugins/carbon_black_defense/plugin.spec.yaml

```diff
@@ -4,9 +4,10 @@ products: [insightconnect]
 name: carbon_black_defense
 title: VMware Carbon Black Cloud Endpoint Standard
 description: Find events and retrieve details for specific events
-version: 2.0.1
+version: 3.0.0
 vendor: rapid7
 support: rapid7
+supported_versions: ["04/02/2022"]
```

Comment:
Kindly update the supported version to March

Reply:
Updated version to current date.